modules/HapticGlove_module/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2020 Fondazione Istituto Italiano di Tecnologia (IIT)
+# All Rights Reserved.
+# Authors: Kourosh Darvish <kourosh.darvish@iit.it>
+
+# set target name
+set(EXE_TARGET_NAME HapticGloveModule)
+
+
+# set cpp files
+set(${EXE_TARGET_NAME}_SRC
+  src/main.cpp
+  src/RobotController.cpp
+  src/RobotInterface.cpp
+  src/HapticGloveModule.cpp
+  src/GloveControlHelper.cpp
+  src/KalmanFilter.cpp
+  src/MotorEstimation.cpp
+  src/RobotMotorsEstimation.cpp
+  src/Retargeting.cpp
+  src/LinearRegression.cpp
+  src/GloveWearable.cpp
+  src/Logger.cpp
+  src/Teleoperation.cpp
+  src/ControlHelper.cpp
+  )
+
+# set hpp files
+set(${EXE_TARGET_NAME}_HDR
+  include/RobotController.hpp
+  include/RobotInterface.hpp
+  include/HapticGloveModule.hpp
+  include/GloveControlHelper.hpp
+  include/KalmanFilter.hpp
+  include/MotorEstimation.hpp
+  include/RobotMotorsEstimation.hpp
+  include/Retargeting.hpp
+  include/LinearRegression.hpp
+  include/GloveWearable.hpp
+  include/Logger.hpp
+  include/Teleoperation.hpp
+  include/ControlHelper.hpp
+  )
+
+# add include directories to the build.
+include_directories(
```

Comment:
Rather than using ``include_directories``, it is better to include ``target_include_directories``. See https://stackoverflow.com/questions/31969547/what-is-the-difference-between-include-directories-and-target-include-directorie

Reply:
agree, fixed.